x/compliance/keeper/msg_server_certify_model.go

```diff
@@ -97,6 +97,13 @@ func (k msgServer) CertifyModel(goCtx context.Context, msg *types.MsgCertifyMode
 	// store compliance info
 	k.SetComplianceInfo(ctx, complianceInfo)
 
+	deviceSoftwareCompliance, _ := k.GetDeviceSoftwareCompliance(ctx, msg.CDCertificationId)
```

Comment:
We need to remove the compliance info from the list in case of revocation.

Reply:
Done! ✅